src/Common/src/TypeSystem/IL/ILImporter.cs

```diff
@@ -304,13 +313,16 @@ private void MarkBasicBlock(BasicBlock basicBlock)
 
         private void ImportBasicBlock(BasicBlock basicBlock)
         {
-            _stackTop = 0;
+            _stack.Clear();
```

Comment:
Actually looking at the code, ILImporter and ILToCppImporter are very much tight together. If there is another ILImporter variation somewhere (none that I can find in the repo), we can either merge both files into one, or made ILImporter an abstract class. If we do not do anything, it seems that it is better to leave things as they are, i.e a direct dependence in ILImporter of data defined/used in ILToCppImporter.